### PR TITLE
scheduler with stub methods. oo and config

### PR DESCRIPTION
**Pull request by pnuckowski**

### CODE REVIEW

File: alamo_scheduler/config.cfg [hunk 1]
```diff
@@ -0,0 +1,15 @@
+[default]
+verbose = True
+
+[redis]
+host = localhost
+port = 6379
+db = 0
+
+[cache]
+update_key = update_required
```
Comment (h00dy) [inline alamo_scheduler/config.cfg]: Why do we need this?

Comment (pnuckowski) [inline alamo_scheduler/config.cfg]: configurable cache key used to let to know scheduler that checks definitions are changed.

File: setup.cfg [hunk 2]
```diff
@@ -12,3 +12,7 @@ packages =
 all_files = 1
 build-dir = docs/build
 source-dir = docs/source
+
+[entry_points]
+console_scripts =
+  alamo-scheduler = alamo_scheduler.manage:main
```
Comment (h00dy) [inline setup.cfg]: no newline at the end ;)

**Comment by h00dy** [comment]
:+1:

**Comment by ch40s3k** [comment]
+1

**Pull request merged**